<html>

<head>
<style type="text/css">
.knitr .inline {
  background-color: #f7f7f7;
  border:solid 1px #B0B0B0;
}
.error {
	font-weight: bold;
	color: #FF0000;
}
.warning {
	font-weight: bold;
}
.message {
	font-style: italic;
}
.source, .output, .warning, .error, .message {
	padding: 0 1em;
  border:solid 1px #F7F7F7;
}
.source {
  background-color: #f5f5f5;
}
.rimage .left {
  text-align: left;
}
.rimage .right {
  text-align: right;
}
.rimage .center {
  text-align: center;
}
.hl.num {
  color: #AF0F91;
}
.hl.str {
  color: #317ECC;
}
.hl.com {
  color: #AD95AF;
  font-style: italic;
}
.hl.opt {
  color: #000000;
}
.hl.std {
  color: #585858;
}
.hl.kwa {
  color: #295F94;
  font-weight: bold;
}
.hl.kwb {
  color: #B05A65;
}
.hl.kwc {
  color: #55aa55;
}
.hl.kwd {
  color: #BC5A65;
  font-weight: bold;
}
</style>


<!-- #begin.rcode results='hide'
knit("./ASA_Poster_Content/content_1.Rhtml", quiet = TRUE)
end.rcode-->

<script type="text/javascript">
  
  (function() {
    // http://stackoverflow.com/questions/10964966/detect-ie-version-in-javascript
    function isIE () {
      var myNav = navigator.userAgent.toLowerCase();
      return (myNav.indexOf('msie') != -1) ? parseInt(myNav.split('msie')[1]) : false;
    }
    
    if (isIE() === 8 || isIE() === 9){
      // Set an absolute base href.
      baseURL = document.URL;
      baseURL = baseURL.substring(0, baseURL.lastIndexOf('/'));
      document.getElementsByTagName('base')[0].setAttribute('href', baseURL + '/_w_83ad5705/');
    }
  })();
</script>


<title>Relative frequency measurements: Metrics for sample quality, sequencing integrity, and batch effects in targeted NGS.</title>
<link rel="stylesheet" type = "text/css" href="bootstrap.css">
</head>
<style>
  .black_overlay{
    display: none;
    position: absolute;
    top: 0%;
    left: 0%;
    width: 100%;
    height: 100%;
    background-color: black;
    z-index:1001;
    -moz-opacity: 0.8;
    opacity:.80;
    filter: alpha(opacity=80);
}
.white_content {
    display: none;
    position: absolute;
    top: 10%;
    left: 10%;
    width: 80%;
    height: 80%;
    padding: 16px;
    border: 16px solid orange;
    background-color: white;
    z-index:1002;
    overflow: auto;
}
</style>

<body>

<h1 align="center"> Relative frequency measurements: Metrics for sample quality, sequencing integrity, and batch effects in targeted NGS.</h1>
<h5 align="center"> Bonnie LaFleur<sup>1</sup>, Dominic LaRoche<sup>1,2</sup>, Kurt Michels<sup>1,2</sup>, Shripad Sinari<sup>2</sup>,  Dean Billhiemer<sup>2</sup></h3>
<h6 align="center"> <sup>1</sup>HTG Molecular Diagnostics, Inc.,  <sup>2</sup>University of Arizona, Tucson AZ</h3>

<!---Main Contents-->
<div class="row">
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-2"><div class="rcode"><!--html_preserve-->

<h4>Next-Generation Sequencing Data is Compositional</h4>
    <p style="font-size:10px">It is broadly accepted that next-generation sequencing measurements provide only <i>relative</i> abundances of transcripts (Robinson and Smyth 2007; Anders and Huber 2010; Robinson and Oshlack 2010; Law et al. 2014; Lovell et al. 2015). The relative nature of next-generation sequencing has important consequences for downstream analyses.</p>
    <ul style="font-size:10px">
      <li>There are several reasons NGS data are relative.</li>
          <ol type="I">
            <li>Finite sample volume from which DNA/RNA is extracted</li>
            <li>Finite number of reads available on each platform</li>
            <li>Read depth (number of reads allocated to a sample) varies from run-to-run and sample-to-sample</li>
          </ol>
      <li>The number of reads for a given transcript is affected by a number of experiment level factors including (Sims et al. 2014):</li>
        <ol type="I">
          <li>Read depth of the sequencing run (sequencing platform)</li>
          <li>Number of samples included in a sequencing run</li>
          <li>Volume of sample input</li>
        </ol>
      <li>Relative abundance implies data arises as a composition</li>
        <ol type="I">
            <li>Data are often given as reads per kilobase per million reads (RPKM) or counts per million (CPM) to account for differences in read depth.</li>
            <li>More reads allocated to any one transcript results if fewer reads allocated to all other transcripts</li>
            <li>Induces a dependency in the data since as one component increases, the other must decrease (Fig. 1).</li>
        </ol>
      
    </ul>
<p align="right"> <a href = "javascript:void(0)" onclick = "document.getElementById('content1').style.display='block';document.getElementById('fade').style.display='block'">Expand</a></p>



<!--/html_preserve--></div></div>
    </form>
  </div>
  
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-3"><div class="rcode"><!--html_preserve--><h4>Challenges of Compositional Data</h4>
<p style="font-size:10px">Relative abundance data is subject to an arbitrary sum constraint, \(\sum_{i=1}^{D} t_i = \mathbb{T} \), where \(t_i\) gives the number of reads to sample \(i\) in a sequencing run with \(D\) samples.</p>
  <ol type="I" style="font-size:10px">
    <li>The sum contraint, \(\mathbb{T}\), changes between platforms, sequencing runs, and samples.</li>
    <li>This leads to an arbitrary difference in <i>scale</i> between samples which various normalization methods have sought to correct.</li>
  </ol>
<div class="row">
  <div class="col-sm-6">    
  <img src="BarAnimation.gif" alt="HTML5 Icon" style="width:200px;height:120px;"> 
  </div>
  <div class="col-sm-6">
   <p style="font-size:10px">
   <b>Figure 1:</b> This animation shows the impact of changes to a single transcript on the relative abundance of the other transcripts being measured. Ignoring the constraints imposed by compositional data can lead to spurious results.
   </p>
  </div>
</div>
<div class="row">
  <div class="col-sm-6">
  <!--<ol type="I" style="font-size:10px">
    <li>Complicated covariance structure</li>
  </ol>-->
    <img src="ASA_Poster_Content/SimplexGeometry.png" style="width:200px;height:75px>">
  </div>
  <div class="col-sm-6">
    <p style="font-size:10px">Figure 2. Compositions lie in the \(D-1\) dimensional simplex (\(S^{D-1}\))</p>
<p> <a href = "javascript:void(0)" onclick = "document.getElementById('content1').style.display='block';document.getElementById('fade').style.display='block'">Expand</a></p>
  </div>
</div><!--/html_preserve--></div></div>
    </form>
  </div>
  
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-4"><div class="rcode"><!--html_preserve--><h4>Counts per Million: A compositional operation</h4><!--/html_preserve--></div></div>
    <p>   <a href = "javascript:void(0)" onclick = "document.getElementById('content3').style.display='block';document.getElementById('fade').style.display='block'">Expand</a></p>
    </form>
  </div>
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-5"><div class="rcode"><!--html_preserve--><h4>Compositional Operations and their Properties</h4><!--/html_preserve--></div></div>
    <p>   <a href = "javascript:void(0)" onclick = "document.getElementById('content4').style.display='block';document.getElementById('fade').style.display='block'">Expand</a></p>
    </form>
  </div>
</div>

<div class="row">
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-6"><div class="rcode"><!--html_preserve--><h4>Composition-based Quality Control</h4><!--/html_preserve--></div></div>
    <p> <a href = "javascript:void(0)" onclick = "document.getElementById('content5').style.display='block';document.getElementById('fade').style.display='block'">Expand</a></p>

    
    </form>
  </div>
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-7"><div class="rcode"><!--html_preserve--><h4>Sub-compositions and Amalgamations</h4><!--/html_preserve--></div></div>
    <p> <a href = "javascript:void(0)" onclick = "document.getElementById('content6').style.display='block';document.getElementById('fade').style.display='block'">Expand</a></p>
    </form>
  </div>
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-8"><div class="rcode"><!--html_preserve--><h4>Example Analysis</h4><!--/html_preserve--></div></div>
    <p> <a href = "javascript:void(0)" onclick = "document.getElementById('content7').style.display='block';document.getElementById('fade').style.display='block'">Expand</a></p>
    </form>
  </div>
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-9"><div class="rcode"><!--html_preserve--><h4>Discussion and Future Directions</h4><!--/html_preserve--></div></div>
    <p> <a href = "javascript:void(0)" onclick = "document.getElementById('content8').style.display='block';document.getElementById('fade').style.display='block'">Expand</a></p>
    </form>
  </div>
</div>

<!--Expanded Contents-->
<div id="content1" class="white_content">
<div class="chunk" id="unnamed-chunk-10"><div class="rcode"><!--html_preserve-->

<h4 align="center">Next-Generation Sequencing Data is Compositional</h4>
<div class="col-sm-4">
  <form class="well" style="height:800px;">
    <h4>Background</h4>
    <p style="font-size:10px">It is broadly accepted that next-generation sequencing measurements provide only <i>relative</i> abundances of transcripts (Robinson and Smyth 2007; Anders and Huber 2010; Robinson and Oshlack 2010; Law et al. 2014; Lovell et al. 2015). The relative nature of next-generation sequencing has important consequences for downstream analyses.</p>
    <ul style="font-size:10px">
      <li>NGS technologies produce <i>relative abundances</i> of transcripts</li>
        <ol type="I">
          <li>Finite sample volume from which DNA/RNA is extracted</li>
          <li>Finite number of reads available on each platform</li>
          <li>Read depth varies from run-to-run and sample-to-sample</li>
        </ol>
      <li>Relative abundance implies data arises as a composition</li>
        <ol type="I">
            <li>More reads allocated to a transcript results if fewer reads allocated to other transcripts</li>
            <li>Induces a dependency in the data (Fig. 1)</li>
        </ol>
    </ul>
    
  </form>
</div>
<div class="col-sm-4">
  <form class="well">
    <h4>NGS Workflow</h4>
    <img src="ASA_Poster_Content/NGS_Workflow.png" style="width:400px;height:400px;">
  </form>
</div>
<div class="col-sm-4">
  <form class="well">
  </form>
</div>


<!--/html_preserve--></div></div>
    <a href = "javascript:void(0)" onclick = "document.getElementById('content1').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
</div>

<div id="content2" class="white_content">
    This is the expanded Content 2. 
    <a href = "javascript:void(0)" onclick = "document.getElementById('content2').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
</div>

<div id="content3" class="white_content">
    This is the expanded Content 3. 
    <a href = "javascript:void(0)" onclick = "document.getElementById('content3').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
</div>

<div id="content4" class="white_content">
    This is the expanded Content 4. 
    <a href = "javascript:void(0)" onclick = "document.getElementById('content4').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
</div>

<div id="content5" class="white_content">
    This is the expanded Content 5. 
    <a href = "javascript:void(0)" onclick = "document.getElementById('content5').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
</div>

<div id="content6" class="white_content">
    This is the expanded Content 6. 
    <a href = "javascript:void(0)" onclick = "document.getElementById('content6').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
</div>

<div id="content7" class="white_content">
    This is the expanded Content 7. 
    <a href = "javascript:void(0)" onclick = "document.getElementById('content7').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
</div>

<div id="content8" class="white_content">
    This is the expanded Content 8. 
    <a href = "javascript:void(0)" onclick = "document.getElementById('content8').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
</div>
<div id="fade" class="black_overlay"></div>


<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
