

<h4>Next-Generation Sequencing Data is Compositional</h4>
    <p style="font-size:10px">It is broadly accepted that next-generation sequencing measurements provide only <i>relative</i> abundances of transcripts (Robinson and Smyth 2007; Anders and Huber 2010; Robinson and Oshlack 2010; Law et al. 2014; Lovell et al. 2015). The relative nature of next-generation sequencing has important consequences for downstream analyses.</p>
    <ul style="font-size:10px">
      <li>There are several reasons NGS data are relative.</li>
          <ol type="I">
            <li>Finite sample volume from which DNA/RNA is extracted</li>
            <li>Finite number of reads available on each platform</li>
            <li>Read depth (number of reads allocated to a sample) varies from run-to-run and sample-to-sample</li>
          </ol>
      <li>The number of reads for a given transcript is affected by a number of experiment level factors including (Sims et al. 2014):</li>
        <ol type="I">
          <li>Read depth of the sequencing run (sequencing platform)</li>
          <li>Number of samples included in a sequencing run</li>
          <li>Volume of sample input</li>
        </ol>
      <li>Relative abundance implies data arises as a composition</li>
        <ol type="I">
            <li>Data are often given as reads per kilobase per million reads (RPKM) or counts per million (CPM) to account for differences in read depth.</li>
            <li>More reads allocated to any one transcript results if fewer reads allocated to all other transcripts</li>
            <li>Induces a dependency in the data since as one component increases, the other must decrease (Fig. 1).</li>
        </ol>
      
    </ul>
<p> <a href = "javascript:void(0)" onclick = "document.getElementById('content1').style.display='block';document.getElementById('fade').style.display='block'">Expand</a></p>




