%Notes from study section:  DIdn't talk about how their method will extend relative sensitivity to count data and how will this method be different from proteomics procedure.  Didn't talk about rationale for useing bayes approach.  How will bioinformatics pipeline will affect the results- need to talk about these will impact the results. Differences of 4 systems should be outlined and how this might affect the downstream analysis.  Need a lot more detail.  It is unclear if the normalization procedures will perform equivalently.  The data sets are not well defined and how the data will be created?  More detail about how the data sets will be aquired or generated.  Can you incorporate some true knowledge to verify that the result will get you closer to the truth????


\documentclass[11pt]{article}
\usepackage{helvet}
\renewcommand{\familydefault}{\sfdefault}
\usepackage[top=.7in, bottom=.5in, left=.5in, right=.5in]{geometry}
\usepackage{wrapfig}
\usepackage{placeins}
\usepackage{fancyhdr}
\pagestyle{fancy}
\fancyhf{}
\rhead{Dominic D LaRoche}
\lhead{Novel methodology for evaluation of next-generation sequencing measurements.}
\cfoot{Page \thepage}
\setlength{\headsep}{0.1in}

\usepackage[colorlinks=true, linkcolor=blue, citecolor=blue]{hyperref}
\usepackage[backend=bibtex,style=authoryear,backref=true,hyperref=true]{biblatex}
\addbibresource{mybib.bib}
\usepackage{amsmath}

%\usepackage[document]{ragged2e}%remove left justification

\begin{document}

\noindent\textbf{Investigator:} Dominic D. LaRoche\\
\noindent\textbf{Proposal Title:} Novel methodology for evaluation of next-generation sequencing measurements.\\

The measurement of RNA expression in biological samples is a fruitful area of study in many areas of biology. NGS-based methods are increasing in use but the errors associated with NGS-based measurements are not well understood.  All existing NGS-based measurement methods involve multiple steps, from sample preparation to data normalization, and it is unclear how errors propagate through the process to affect the final measurement. This information is clearly needed to improve the accuracy of measurements and speed the pace of discovery.\\

No methods exist to directly compare the accuracy and precision of NGS based RNA measurement systems or to evaluate the sources error.
\textbf{\emph{We hypothesize that relative sensitivity can be adapted to provide a novel method for evaluating the error associated with next-generation sequencing technologies.}}  
Relative sensitivity was originally developed by John Mandel in 1984 for evaluation of measurements in analytical chemistry.  The method has been largely ignored in the evaluation of modern methods but has several useful properties. For one, relative sensitivity is invariant to differences in scale and monotone transformations. This makes it an ideal candidate for comparing measurements that utilize different scales or data transformations, such as RNA-seq. We believe the theory of relative sensitivity will provide a powerful framework for comparing measurement methodologies without the need for individual calibration curves or known concentrations of RNA sequences.\\
 
\noindent\textbf{Aim 1: Develop relative sensitivity  method for NGS-based count data.}
We will reformulate the original relative sensitivity model to accommodate the count data associated with NGS measures.    Since the true distribution of NGS count data is unknown we will re-formulate the relative sensitivity model using an empirical approach as well as a Bayesian approach using three popular distributional forms for RNA sequence data: 1) Poisson, 2) Gamma-Poisson mixture (negative-binomial), and 3) zero-inflated Poisson.  We will then assess the utility of relative sensitivity using simulated RNA-seq data from all three generating models.  This will enable us to characterize the bias and error around the relative sensitivity measure and determine which model formulation: 1) is most robust to violations of its underlying distributional assumption, 2) provides the best estimates of precision for high, medium, and low expressing probes, and 3) has the most stable estimates over the entire range of measurement.\\

\noindent\textbf{Aim 2: Evaluate popular normalization procedures.} It is well understood that NGS based measurements must be normalized prior to analysis.  However, there are currently many normalization methods available and it is unclear which method is optimal. The only published comparison of normalization methods evaluated normalization based on the number of differentially expressed probes found in the final analysis. Not only was this result specific to differential expression analyses, it was also of limited utility due to the coarse nature of the outcome measure.  We will use the estimated relative sensitivity measure to compare 6 popular normalization methods on both simulated data (from all three generating distributions above) and NGS data generated from technical replicates of universal RNA.  Using relative sensitivity to compare these methods will uncover the relative ways each method propagates or mitigates measurement error and batch effects.\\

We will create an R package that provides comprehensive functions to handle NGS measurement data and implement our methods.  We will also create a free website to create a user friendly and secure  graphical user interface for users to easily upload data and implement our methods.  We believe a website provides the most accessible form of methodological implementation to a broad scientific audience. \\

Future advancements in the use of measured RNA for understanding biological systems depends on reliable measurements, particularly as the effect sizes researchers are looking for decrease.  The necessary first step towards reliable measurements is understanding where errors arise and what factors influence them. Our proposed method will enable researchers to directly and easily compare the precision of competing NGS based measurement systems (or components thereof). We will provide accessible tools to evaluate the error of each step in the measurement process such as the sample preparation method or the normalization method. our proposed method of evaluation will not require the use of samples with known RNA sequence counts, which is currently required and prohibitively difficult. 

\section{Significance}
The accuracy and precision of many RNA measurement systems is largely unknown despite the importance of these measurements in modern biological research. We define an RNA measurement system as any sequence of procedures designed to quantify RNA from a biological sample and provide data for analysis.  There are currently over 5 widely used RNA measurement systems (4 of which utilize next-generation sequencing technology), each with a different form of technology to achieve the final measurement.  Our proposed research will provide a methodology for studying measurement error in NGS-based RNA measurement systems.  Specifically, we will provide a method to directly compare the precision of any two RNA measurement systems using only technical replicates from each system. Even though RNA is widely studied, the precision of measurements may limit the quality of inference and slow the rate of scientific progress. Continued progress in fields that rely on NGS-based RNA measurement systems will depend on understanding how and why errors occur.\\

There is currently no robust methodology for comparing measurement precision between NGS-based RNA measurement systems.  The current approach to comparing measurement systems is to indirectly compare each system's reproducibility and repeatability, an approach with limited utility (\cite{Lovell2015}).  Alternatively, others have used the final outcome measure, i.e. differential expression of genes, as a measure of performance. However, the outcome of an analysis is only a crude measure of measurement precision at best, even when using simulated data where the truth is known.  Our method will allow direct comparison of the precision of any two measurement systems.  This will allow for: 1) evaluating measurement systems for diagnostic procedures; 2) improving a single measurement system through experimentation to determine where errors arise; 3) evaluating the relative impact on precision from the numerous data normalization methods currently in widespread use; and 4) selecting the best measurement system and normalization method for any given experiment. Finally, we will provide all of our methodology as a freely accessible web application.\\

\section{Innovation}

We will apply a little known framework, Relative Sensitivity, originally developed by John Mandel in 1984 for evaluating measurement in analytical chemistry.  This method does not require knowledge of the true amount of the sequences being measured, a quantity that is often difficult or impossible to know in RNA measurement problems.  Relative sensitivity provides the framework for making comparisons among normally distributed measurements.  Our approach is conceptually innovative because we will need to derive the formulas necessary to apply relative sensitivity to less well-understood count data, often with multiplicative errors, arising naturally from NGS-based measurement systems.\\

A new methodology is only useful if it is accessible to a wide range of researchers and scientists.  However, many advances in statistical methodology take considerable time for adoption by the general scientific community because of the barriers to implementation. Our approach is technically innovative because, if we find utility in our proposed methods, We will provide a software suite for implementing our methods through the recently developed web platform for R programs, shinyapps.io.  This platform allows users to interact with statistical analyses pre-programmed in R through a web-based graphical user interface (GUI).  The platform greatly reduces the barriers to method implementation because it does not require anything to be installed on user computers and can provide elaborate point-and-click documentation to guide the user through the analyses. We will provide our suite of methods as a dynamic GUI available for free to researchers who wish to compare measurement or normalization systems on their own data.  This platform has never been used for widespread implementation of a new method, but holds great potential as a translational statistics tool.\\
%temper this paragraph with an if the product is useful statement.

\section{Specific Aim 1 Research Plan}

\subsection{Background and Rationale}
There is no established method for directly comparing the precision and accuracy of NGS-based RNA measurement systems.  The rapid adoption of NGS-based RNA measurement systems across a wide range of biological disciplines necessitates improved understanding of the precision of these systems.  There are at least four different measurement systems which utilize next generation sequencing technology currently in widespread use.  These systems all have different technological features and are unlikely to produce identically precise results.  The ability to compare measurement systems would also enable scientists to select the optimal system for their experiment and allow manufacturers to identify sources of error and make improvements. Currently, measurement systems are evaluated on the basis of repeat-ability and reproducibility but there is no way to directly compare existing metrics of repeat-ability and reproducibility among competing measurement systems due to the differences in scales and underlying distributions of the measurements.  \\%something about proportional data here

The objective of this aim is to establish a methodology to directly compare two measurement systems.  We hypothesize that the method of relative sensitivity will provide a powerful and useful framework for making these comparisons.  The theory of relative sensitivity was originally developed by John Mandel in 1984 (\cite{JohnMandel1984}) as an extension of his original work on the sensitivity of analytical chemistry measurements (\cite{Mandel1957}).  Relative sensitivity has several key properties which make it suitable for this application.  For example, relative sensitivity is not affected by the scale of the measurement, which can vary substantially between NGS-based measurement systems.  Moreover, most measurement systems include some monotone transformation of the final result for analysis, of which there are many choices, and relative sensitivity is invariant to these transformations.\\

We will use both simulated and real data from 4 measurement systems to evaluate the estimates of precision and accuracy we generate from the relative sensitivity framework.  Simulated data is necessary as the truth of real-world measurements can never be known.  We will focus on simulating the measurement process, rather than the final result as is typically done, so that we can understand the impacts of each step in the measurement process.  We will confirm the utility of our method by evaluating the accuracy and precision of four competing measurement systems: HTG EdgeSeq, NanoString nCounter, Illumina RNASeq, and TaqMan Gene Expression assays. We will use technical replicate samples and quantify each sample replicate 24 times%state that this is arbitrary
on each platform.  We have already completed sample quantification on HTG EdgeSeq and NanoString nCounter. \\

A metric for comparing NGS-based measurement systems will have immediate utility in identifying optimal measurement systems and sources of measurement error.  This method will provide a critical window into the performance of measurement systems for both consumers of these systems and the manufacturers of these systems.  Due to the diverse nature of the technology of systems currently in use, it is likely that the performance of these systems will correlate with elements of the technology utilized. We believe understanding these differences in performance will be important for mitigating problems at the development level and selecting the appropriate method for a given experiment.\\

At the completion of this aim we expect that we will provide an implemented methodology for comparison of any 2 NGS-based measurement systems such that a scientist with little understanding of the underlying theory of relative sensitivity will be able to implement the methodology and interpret the results.\\

\subsection{Experimental Plan}
The theory of relative sensitivity was initially presented in John Mandel's book \emph{The Statistical Analysis of Experimental Data} (1984) but was never widely adopted in the analysis of measurement systems outside of analytical chemistry (with few references even within analytical chemistry).  We attribute this to the relatively spotty uptake of statistical procedures in general and to the limited utility of such a method when calibration curves are more easily attainable.  However, we believe the theory of relative sensitivity provides a simple, yet powerful, statistical framework for the evaluation of complicated NGS-based measurement systems.  \\


\begin{figure}
\center
<<SensFig, echo=FALSE, message=FALSE, warning=FALSE, fig.height=3, fig.width=5>>=
library(ggplot2)
df <- data.frame(x=seq(-5,0, by=.01))
df$y <- ifelse(df$x <=0, y=dnorm(df$x)*10, 4)
df$x <- df$x+5
df$epsilon <- .5-(df$x/5-(df$x/5)^3)
df2 <- data.frame(y=dnorm(seq(-6,-1, by=.01), sd=3)*30, x=seq(-5,0, by=.01)+5)
df2$epsilon <- (1/(df2$y+1))*.5
ggplot(df, aes(x=x, y=y)) + geom_path() + ylab("Measured Value") + xlab("True Value") + theme_bw()
@
\caption{\small A single sensitivity curve for a measurement system representing the relationship $Y = f(X)$, where $f(X)$ is some unknown non-linear function of $X$.  The sensitivity of the measurement is greatest between the true values of 3 and 4.5 and smallest at 0 and 5.  This measurement system would not be a good choice for those interested in measuring true values of less than 2 since the method is unable to differentiate between these values.}
\label{sensslope}
\end{figure}


The theory of relative sensitivity is an extension of Mandel's work on estimating sensitivity curves (1957) which removes the need to know the actual analyte concentration in order to evaluate the precision of a measurement. The sensitivity of a measurement is the slope of the functional relationship between a property of interest and its measurement,  $Y = f(X)$,  where $Y$ is the measured value, $X$ is the true value, and $f(X)$ represents some unknown relationship between these values.  The function $f(X)$ maps a  can be non-linear and/or non-increasing (or decreasing) in practice but must be monotonic to be useful.  If the true count of target RNA is known for each sample and a variety of samples are measured, each with a different amount of target RNA, then a calibration curve can be constructed as in figure~\ref{sensslope}.\\

For a given measurement system, steep slopes ($f(X)$) correspond to greater sensitivity because a steep slope results in large differences in the measured value for small differences in the corresponding property being measured (fig.~\ref{sensslope}).  The error around a sensitivity curve for a measurement process also affects the utility of the measurement.  For example, if the slope is small but there is little error then the measurement can still discriminate between different states of the property whereas large error can ``swamp out" a steeper slope (fig.~\ref{senserr}).\\



\begin{figure}
\center
<<SensErrFig, echo=FALSE, message=FALSE, warning=FALSE, fig.height=3, fig.width=6>>=

df$set <- "Measurement 1"
df2$set <- "Measurement 2"
dfall <- rbind(df, df2)
ggplot(dfall, aes(x=x, y=y)) + geom_path() + geom_ribbon(aes(ymax=y+epsilon, ymin=y-epsilon), alpha=.2) + ylab("Measured Value") + xlab("True Value") + facet_wrap(~set, ncol=2) + theme_bw()
@
\caption{\small Hypothetical calibration curves for 2 measurement systems.  The shaded region around each line represents the standard error of the measurement over the range of true values.  A steeper slope allows better discrimination between different levels of the true value.  Likewise, small errors are also better. If these calibration curves were known there would be no need for the relative sensitivity method as one could simply compare the slopes and errors of the two curves and identify measurement 1 as superior to measurement 2 for true values between 2.75 and 4.5.  Unfortunately, these curves cannot be constructed without knowing the true value for each measured value.}
\label{senserr}
\end{figure}


In order to construct and evaluate a sensitivity curve as described above one must know the true value of the property being measured.  For many NGS-based measurement systems this property is either unknowable or is exceedingly difficult to know.  However, relative sensitivity can be used to compare two measurement systems without knowing the true state of the underlying property being measured.  A key insight of Mandel's was that two measurements of the same quantity must be related.  Mandel used this insight to construct the  relative sensitivity curve as a way to compare two measurement systems, resulting in measurements $Y_1$ and $Y_2$, without knowing the true value of the measured quantity.  The relative sensitivity ratio takes the form:

$$RS(Y_1/Y_2) = \frac{\left| dY_1/dY_2 \right|}{\sigma_{Y_1}/\sigma_{Y_2}}.$$%be consistent with model terminology above

In this formulation, $dY_1/dY_2$ represents the slope of the relationship between $Y_1$ and $Y_2$, i.e. the slope of the line in a plot of $Y_1$ versus $Y_2$, while $\sigma_{Y_1}$ and $\sigma_{Y_2}$ represent the standard deviations of the two measurement systems.  This ratio does not involve the true value of the property being evaluated while providing a simple metric of relative utility for two measurements.  For most measurement systems the sensitivity of the measurement is not linear (fig.~\ref{sensslope}) and, therefore, the slope of the relationship between measurements $Y_1$ and $Y_2$ will also vary (fig.~\ref{relY1Y2}).  The resulting changes in the value of the relative sensitivity ratio produce the relative sensitivity curve (fig.~\ref{relsens}).  When the relative sensitivity is less than 1 method 2 is better whereas when the relative sensitivity is greater than 1 method 1 is better.  From figure~\ref{relsens} we can see that method 2 outperforms method 1 until the true value of the measured quantity is about 2.75, at which point method 1 becomes preferable until the true measurement is about 4.5.  If an experimenter expected the true range of measurements to lie above 2.5 and below 4.5 then method 1 would be clearly superior to method 2, whereas if the expected value of the true measurements were all below 2.5 then method 2 would be preferable.\\

\begin{figure}
\center
<<relY1Y2, echo=FALSE, message=FALSE, warning=FALSE, fig.height=3, fig.width=5>>=
dfr <- data.frame(Y1 = df$y, Y2 = df2$y, truth = df$x, sd1 = df$epsilon, sd2 = df2$epsilon)
#calculate slope
dfr$dy1 <- NA
dfr$dy2 <- NA
for(i in seq_along(dfr$Y1)[-1]){
  dfr$dy1[i] <- dfr$Y1[i] - dfr$Y1[i-1]
  dfr$dy2[i] <- dfr$Y2[i] - dfr$Y2[i-1]
}
dfr$dy1dy2 <- dfr$dy1/dfr$dy2
ggplot(dfr, aes(x=truth, y=dy1dy2)) + geom_path() + ylab("|dY1/dY2|") + xlab("True value of the measured Quantity") + theme_bw()
@
\caption{\small The slope of the relationship between the two measurement systems shown in figure~\ref{senserr}, $dY_1/dY_2$, for each value of the true quantity being measured (this slope must be estimated in practice). }
\label{relY1Y2}
\end{figure}

\begin{figure}
\center
<<relSens, echo=FALSE, message=FALSE, warning=FALSE, fig.height=3, fig.width=6>>=
 dfr$sdr <- (dfr$sd1/dfr$sd2)
dfr$rs <- dfr$dy1dy2/dfr$sdr
ggplot(dfr, aes(x=truth, y=rs)) + geom_path() + geom_hline(yintercept=1) + geom_text(x=0.5, y=0.7, label = "Method Y2 better", size=3.5)  + geom_text(x=3.5, y=1.4, label = "Method Y1 better", size=3.5) + ylab("Relative Sensitivity") + xlab("True value of the measured Quantity") + theme_bw() #+ geom_area(data=subset(dfr, rs>1), fill="red")
@
\caption{\small The relative sensitivity for the two measurement systems resulting in the measured quantities $Y_1$ and $Y_2$.  Here we have the truth plotted along the x axis; in practice this would not be available in which case substituting either $Y_1$ or $Y_2$ would suffice.  When relative sensitivity is $<1$ method Y2 is better whereas when relative sensitivity is $>1$ method 1 is better. Relative sensitivity combines both the relative slope of each measurement (fig.~\ref{relY1Y2}) and the error for the range of true values in the sample used.  The relative sensitivity shown here can be compared to the two measurement calibration curves shown in figure~\ref{senserr}.}
\label{relsens}
\end{figure}
%explain how we will simulate data, use example from literature

To estimate the relative sensitivity, one needs to know the slope of the relationship between the two measurement systems being compared as well as the standard deviations of the two measurements.  The standard deviations of the two measurements can be easily estimated from technical replicates. We will compare two approaches for estimating the relationship between the two measurements.  The original formulation by Mandel suggested directly estimating the slope of the relationship by regressing $y_1$ onto $y_2$.  Unfortunately, this method assumes no measurement error in $y_1$ which is clearly not plausible.  We will, instead, compare two different approaches to account for the measurement error in both measurements. First, for an assay with $n$ probes we will use an empirically estimated slope by calculating the change in each measurement from each rank-ordered probe to the next higher probe, $\delta y_i = y_{ij}-y_{i(j-1)}$ where $i=$1, 2 and $j=$2, ..., $n$. We will then calculate the empirical slope, $s_j$ as the ratio $s_j=\frac{\delta y_{1i}}{\delta y_{2i}}$. Second, we will use a model based Bayesian approach to simultaneously estimate the slope and measurement errors.  RNA sequence data can reasonably be considered to arrive from 3 different distributions:  1) Poisson, 2)
Gamma-Poisson mixture (negative-binomial), and 3) zero-inflated Poisson. 

We will test the utility of our method by simulating the measurement processes under consideration.  Frequently, investigators will simply simulate the final measurement using assumptions which favor the method under investigation.  We will simulate the measurement process for each of the 4 measurement systems under consideration using each of the 3 possible underlying distributions.  This approach will have 3 benefits: 1) we believe the final simulated data set will more closely match real data likely to be encountered by end users, 2) we will be able to investigate how perturbations for a given step within a measurement system affect the final utility of the measurement, and 3) simulated data for each measurement protocol will have arisen from the same true concentration of target RNAs.  We will also test our method using real data collected from all four measurement systems under review.  We have already collected this data for identical samples on the HTG EdgeSeq and the NanoString Ncounter platforms.\\  %more information about where the data comes from.  Where does the data come from how can we bue sure we ave suitable range of data, etc...
%what are the steps for each machine. explain the differences between platforms and how these could lead to variuous differnt rsesults.

We will implement the method in the open source statistical package R and provide a open source package of functions to import data and compare measurement systems.  We will also provide a graphical user interface to this package by creating an interactive web page that allows users to point and click through an analysis.  We will host the application through the free shiny.io service provided by Rstudio.\\

\FloatBarrier
\subsection{Expected Outcomes, Potential Problems, and Alternative Strategies}
At the completion of this aim we expect to have an implemented methodology which can compare any two NGS-based measurement systems in terms of precision and accuracy.  We expect to have our method freely available for use by scientists through a user-friendly graphical user interface available to anyone with an internet connection.  We also believe the FDA will benefit from our methodology due to the increasing use of NGS-based measurement systems to make clinical decisions. \\ 

The utility of relative sensitivity to evaluate subtle differences in the performance of NGS-based measurement systems is obviously unknown and there are several challenges to this proposed methodology.  Specifically, relative sensitivity requires that the two measurement systems are measuring the same quantity.  We will use specially prepared universal reference RNA (Agilent Technologies) to ensure the highest similarity across samples run on different platforms.  We will also ensure that assays developed for different platforms are actually targeting the same RNA sequence.  Moreover, our estimation of the relationship between any two measurement systems requires a broad range of target RNA abundance.  Our previous experience suggests that the combination of universal RNA samples and broad panel of probes for each measurement system will fulfill this requirement.  However, it is possible, even after taking these precautions, that we will be unable to differentiate between existing systems using our proposed estimation of the relative sensitivity curve.  Given this scenario we will still be guaranteed to identify an upper bound on the difference in precision between any two measurements using information obtained through simulations combined with results from real samples.\\

If we are unable to detect a difference in precision between measurement systems using our proposed estimation scheme we will re-formulate our estimators based on the maximum-likelihood.  These estimators require more assumptions about the data but will likely yield smaller confidence intervals around our estimates of precision due to these assumptions.  Although we believe our proposed method with fewer assumptions is more desirable, the assumptions made by the maximum likelihood estimates are common when dealing with NGS-based data so we feel this method would also provide utility to scientists and manufacturers.\\

\section{Aim 2 Evaluation of frequently used normalization procedures.}

\subsection{Background and Rationale}

Data from NGS-based RNA measurement systems must be normalized prior to most subsequent data analyses to remove systematic technical effects.  The normalization process can therefore be considered as part of the measurement process and suitable for investigation with the relative sensitivity method. There are currently over 7 methods to normalize NGS-based measurement data in widespread use and no clear guidance exists for selecting a method.  These methods include: 1) total count, 2) upper quartile (\cite{Bullard2010}), 3) DESeq2 (\cite{Anders2010}), 4) Median, 5) Trimmed mean of M-values (TMM; \cite{Robinson2007}), 6) Quantile (\cite{Amaratunga2001}), and 7) Reads per Kilobase per Million (RPKM; \cite{Mortazavi2008}).  A recent comparison of these 7 popular normalization methods by Dillies et al. (\cite{Dillies2013}) was ambiguous about the optimal normalization method and under what conditions optimality would hold.  Two years after the comparison was published we still see widespread use of nearly all of the methods compared, indicating uncertainty among scientists and statisticians about the optimal method.  At best, researchers are selecting methods based on limited, and sometimes conflicting, information.  At worst, researchers are selecting a normalization method which they find to be optimal in terms of the results they expect (or would like) to see.\\
%incorporate a direct comparison with previous method of evaluating normalization methods. %graphical explanation of the different normalization procedures.  More detail on the Dillies et al paper.  Provide examples about how previous method 

Our aim is to create an objective measure of the relative efficacy of the 7 most popular normalization methods so that researchers can select the optimal method for their data \emph{without} basing this decision on the final outcome.  The normalization procedure can be thought of as part of the measurement system.  As such, we hypothesize that relative sensitivity can be adapted to provide a relative measure of performance for any pair of normalization methods. By comparing the impact of each type of normalization on the relative precision of the measurement we will provide necessary information for researchers when selecting a normalization method.\\

The review article by Dillies et al. utilized both simulated and real data.  For both types of data the authors used the number of differentially expressed genes detected after normalization as the measure of performance for each normalization method.  For the simulated data this is an appropriate, if crude, measure of performance.  However, for the real data since the number of truly differentially expressed probes is unknown this is not appropriate.  The practice of selecting a normalization method based on `optimal' final results could lead to an increase of false discoveries due to the circular nature of the decision.  Moreover, RNA measurements are utilized for more than just differential expression analysis and no comparison of normalization methods has been conducted outside of the differential expression paradigm.  Unfortunately, without some objective measure or clear guidance this practice will likely continue.  Our research will provide an objective measure and guidance about the impacts of all 7 commonly used normalization methods.  We believe this methodology will improve the quality of research across the wide range of research areas which utilize this type of data.\\

\subsection{Experimental Plan}
Existing normalization methods are based on different assumptions about the underlying nature of the observed data.  We will simulate  observed data under a variety of these assumptions in order to compare normalization methods under conditions when assumptions are and are not met.  We will use relative sensitivity to evaluate each normalization method against each simulated data type to determine when each method is either appropriate or inappropriate and optimal or sub-optimal.  We will define a method as being appropriate when the inferences that would be made from normalized data (e.g. differential expression results) are consistent with the truth.  We will define an optimal method as the method normalization which produces final measurements with the highest precision across the entire range of measurement.\\

Some methods may outperform others in only a particular range of measurement.  We will divide the range of measurement into high, medium, and low expressed probes by dividing the simulated measurements into quadrilles.  Low expressed probes will be defined as those less than the first quartile, medium expressed as those between the 1st and 3rd quartiles, and high expressed as those above the 3rd quartiles.  Since different normalization methods are likely to place different sets of probes into each of these categories we will identify which probe is assigned to each category based on the random variable, $\lambda$, associated with the expected count for that simulated probe. \\

Simulations provide important information about the performance of normalization under tightly controlled conditions where the truth is known.  However, real data often do not conform with the simplified and controlled nature of simulations.  Therefore, we will  also compare normalizations on real data using the technical replicate samples quantified on each of the 4 platforms identified in aim 1.  For each measurement platform we will generate 1 set of data from technical replicates of universal RNA (Agilent Technologies).  We will then normalize each data set using all 7 methods listed above.  We will then We will use relative sensitivity to evaluate the effect of each normalization method for the data generated by each platform.  Using this method we will control for platform differences when comparing normalization methods.  We will further assume that differences between normalizations on different platforms are due to the normalization x platform interaction and not differences in samples used for technical replicates.  Our experience with universal RNA from Agilent Technologies supports this assumption.\\


\subsection{Expected Outcomes, Potential Problems, and Alternative Strategies}

Previous research has shown many normalization methods to perform similarly on real observed data (\cite{Dillies2013}; \cite{Rapaport2013}).  Therefore, we expect that the performance, as measured by relative sensitivity, of each normalization method to also be quite similar.  However, we believe relative sensitivity will provide much more information about the precise nature of the differences in normalization methods than previous comparisons.  Previous comparisons of normalization methods relied on using the outcome, e.g. the number  of differentially expressed probes, as the measure of the effect.  The outcome is a very crude measure of the effect of a normalization procedure and has been unable to differentiate among competing procedures in previous research. Relative sensitivity has the potential to directly compare the effects of each normalization method allowing the direct comparison of each method with respect to any, or all, measurement range(s). We believe this level of detail will enable us to differentiate the impacts of closely related normalization methods such as TMM (\cite{Robinson2007}) and Quantile (\cite{Amaratunga2001}) methods. The nature of these differences will highlight when each of these methods is optimal and where each method performs poorly.\\

\newpage
\printbibliography

\end{document}