\documentclass{book}
\usepackage[letterpaper, top=1in, bottom=1.25in, left=1.25in, right=1.25in]{geometry}
\usepackage[colorlinks=true, linkcolor=blue]{hyperref}
\usepackage[backend=bibtex,style=authoryear,backref=true,hyperref=true]{biblatex}
\addbibresource{mybib.bib}
\usepackage{amsmath}

\title{Dissertation Proposal:\\ Methods for the Analysis of Compositonal RNA Sequence Data}
\author{Dominic D LaRoche}

\begin{document}
\maketitle
\tableofcontents

\chapter[Background]{Background and Introduction to the Problem}

\section{RNA Sequencing Data}
%General information about RNA sequence data: where it comes from and how people use it.

\subsection{Process of Collecting RNA Sequence Data}
%How samples are collected, prepared, and sequenced

\subsection{General Properties of RNA Sequence Data}
%What are the consequences of the process and the nature of RNA? i.e. poisson, neg-bin, and sum constraints.  Factors affecting sequencing
%two sum constraints in RNA seq data: one related to the size of the sample (sample bucket) and another related to the number of mapped reads (sequencing depth bucket)


\subsection{Current Methodology Associated with RNA Sequencing Data}


\section{Principles of Compositional Data Analysis}
Compositional data are non-negative data which are subject to a sum constraint, i.e. all the elements must sum to unity.  This simple constraint has some important consequences for many standard statistical methodologies including correlation and regression.  Compositional data contain only relative information, i.e. the information about any individual component, or group of componenets, is relative to the other components and no absolute information about the absolute value of the component.  For example, if we know that 20\% of the food in a refigerator is composed of fruit we do not know how much total fruit there is.  If the refigerator is full then there will be substantially more fruit than if the refigerator is nearly empty.\\

Potential problems associated with compositional data were identified as early as 1897 by Pearson who noted that spurious correlations can be induced through ratios of independent variables, e.g. if $X$, $Y$, and $Z$ are uncorrelated then $X/Z$ and $Y/Z$ will be correlated. Despite the fact that compositional data naturally arises in a wide variety of scientific disciplines, a general method for analysis of compositional data was not developed until John Aitchison published his seminal book in 1986.  Aitchison outlines some basic principles for compostional data analysis (section~\ref{subsec:fund}) and provides some analysis tools for compositional data which conform to these principles (section~\ref{subsec:methods}).  Additional methodology has been developed by a number of authors in the 29 years since the publication of Aitchison's book, although a number of problems remain. 

\subsection{Fundamental Principles}
\label{subsec:fund}
Aitchison outlined a set of fundamental principles to which all methods for compostional data should adhere (\cite{Aitchison1986}).  These principles are outlined below.

\subsubsection{Scale Invariance}
Scale invariance requires that the results of a statistical procedure should not depend on the scale used.  Any meaningful function $x$ of a composition $w$ must satisfy:

$$f(pw) = f(w) \text{, for every }p>0.$$

Aitchison notes that any meaningful (scale-invariant) function of a composition can be expressed in terms of ratios of the components of the composition.  For example, any method used for compositional data should not give different results whether the composition is given as proportions, percentages, parts per million, or any other scale.\\

\subsubsection{Sub-compositional Coherance}
Sub-compositional coherance requires that the results of a statistical procedure on a subset of componenets from a composition should depend only on the data contained in that subset.  A subcomposition is defined as the (1, 2, ..., $C$) parts of a $D$-part composition $\left[x_1, ..., x_D\right]$:

$$\left[s_1, ..., s_c\right]=\frac{\left[x_1, ..., x_c\right]}{\left(x_1 + ... + x_c\right)}$$

Any changes in components $\left[x_{c+1}, ..., x_D\right]$ should not have an impact on the inference from the subcomposition.  For example, if we measure the number of reads of 14 mRNA sequences from a sample that contains 4,000 unique mRNA sequences the inference we obtain from those 14 sequences should not be affected by the expression level of any of the other 4,000 sequences.\\ 

\subsubsection{Permutation Invariance}
Permutation invariance requires that the results of a statistical procedure should not depend on the ordering of the components. 


\subsection{Statistical Methods for Compositional Data}
\label{subsec:methods}
\subsubsection{The Simplex}
Traditional statistics is concerned with making inferences from points in $R^D$.  However, the sample space for compositions is restricted to the \emph{Simplex}, $S^D$ because of the sum constraint.  This fundamental difference in sample space necessitates an alternative methodology and renders inference from traditional regression and correlation meaningless.  Aitchison (1986) developed much of the current methodology for compositional data through careful examination of the algebraic-geometric structure of the simplex.\\

It is typical to transform compostional data to the \emph{unit simplex} by dividing each component by the sum of components such that the sum of transformed components is equal to 1: 

$$C\left[ x_1 ... x_d \right] = \left[ x_1 ... x_d \right]/ \sum_{i=1}^D x_i$$

The notation $C[\cdot]$ is termed the \emph{closure operation} and scales a composition x to the unit simplex.
%use the unit simplex throughout?
There are two fundmental operations frequently used in $R^D$ for which Aitchison defined equivalent operations in the simplex: 1) translation, and 2) scalar multiplication.  \\


\subsubsection{Perturbations}

Aitchison identified the need for an operation in the simplex equivalent to $X=x+t$, the translation $t$. This operation, defined as a \emph{perturbation} and denoted by $\oplus$, takes the form:

$$X = p \oplus x = \frac{\left[ p_1x_1...p_Dx_D \right]}{\left( p_1x_1 + ... + p_Dx_D \right)} = C\left[ p_1x_1...p_Dx_D \right]$$

where, $p$ is the perturbation which translates $x$ to $X$ and $C$ is the \emph{closure} operation which scales the composition to the unit simplex.  The perturbation operator leads to several other useful metrics in the simplex such as the distance between two compositions:

$$.$$

As well as for characterizing the imprecision or error around a measurement:

$$x_n = \Upsilon \oplus p_n  \ \ (n=1, ... , N),$$

where the $x_n$ are the observed measurements, the $p_n$ are independent error perturbations characterizing the imprecision,  and $\Upsilon$ is the true underlying composition.   


\subsubsection{The Power Operation}
Like perturbation is the simplical equivalent of translation, Aitchison defined the power operation as the simplical equivalent of multiplication.  For any real number $a \in R^1$ and any composition $x \in S^D$ the power transform of $x$ is defined as:

$$X = a \otimes x = C \left[x_1^a ... x_D^a \right].$$

The power transformation enables a form for regression between a fixed variable $v$ and a composition $x$:

$$ x= \Upsilon \oplus \left\{ log v \otimes \beta \right\} \oplus p.$$

In this formulation $\beta$ is composition analogous to regression coefficients and $p$ is a perturbation analogous to an error term in regression.

\subsubsection{Logratio Analysis}

In order to satisfy the scale invariance of compositional data it is typical to work on ratios of the components.  Furthermor, since the sample space of ratios of positive numbers is not in the whole of real numbers, it is typcial to work in the logarithms of ratios.  Three popular transformation exist for a $D$-part composition: 1) additive log-ratio, 2) centered log-ratio, and 3) ilr. The alr transformation is defined as:

$$alr(x) = [log(\frac{x_1}{x_D})\ log(\frac{x_2}{x_D})\ ...\ log(\frac{x_{D-1}}{x_D})]$$

and reduces the dimension of the compositional vector from $D \rightarrow D-1$.  
%alr - could this be sensitive to the choice of denominator?  I.e. how much will results be affected if the denominator is a highly variable probe?
%look into 'compositional independence'



\section{Current Methodology with Respect to Compositional Data Analysis}
%The use of log-ratios is widespread in analysis of RNA data but these ratios are often *between* compositions rather than within them.
\subsection{Normalization}
Previous authors have identified the compositional nature of RNA sequencing data (\cite{Robinson2010}).  As stated previously, RNA sequence data are likely subject to two sum constraints: 1) the number of RNA sequences that can fit into the finite sample collected, and 2) the number of available reads of those sequences for the given sequencing technology.  %can both of these exist simultaneously? 
\subsubsection{Median Normalization}


\subsubsection{Trimmed Mean of M-values Normalization Method}
Robinson and Oshlack (2010) primarily focused on the mapped read constraint when developing their Trimmed-Mean of M-values (TMM) normalization method for RNA sequence data.  Like many others (\cite{Anders2010}), they also assume that the majority of genes in an assay are not differentially expressed.  For sequencing data Robinson and Oshlack first define the gene-wise log-fold-changes as:

$$M_g = log_2 \frac{Y_{gk}/N_{k}}{Y_{gk\prime}/N_{k\prime}},$$

where $Y_{gk}$ is the read count for gene $g$ in sample $k$ (here we assume 1 library for each sample) and $N_k$ as the total number of reads for sample $k$.  They then define the absolute expression levels as:

$$A_g = \frac{1}{2}log_2\left(Y_{gk}/N_k \cdot Y_{gk\prime}/N_{k\prime} \right)\text{ for }Y_{g\bullet} \ne 0,$$

assuming the absolute expression level is the same for the two samples.  Both the $M$-values and $A$-values are timmed removing the upper and lower 30\% of the $M$-values and the upper and lower 5\% of the $A$-values (although these values are defaults and can be tailored for a given experiment).  The resulting normalization factor for sample $k$ is then a weighted average of the $M$-values:

$$log2\left( TMM_k^r \right) = \frac{ \sum_{g \in G} w^r_{gk}M^r_{gk}}{\sum_{g \in G} w^r_{gk}}$$

where

$$M^r_{gk} = log_2\frac{Y_{gk}/N_{k}}{Y_{gr}/N_{r}}, \ Y_{gk}, Y_{gr} > 0\text{, for reference sample }r, $$

and the weights, $w^r_{gk}$, are defined as:

$$w^r_{gk} = \frac{N_k-Y_{gk}}{N_kY_{gk}} + \frac{N_r-Y_{gr}}{N_rY_{gr}}, \text{ for } Y_{gk}, Y_{gr} > 0.$$

The weights are a result of taking the inverse of the approximate asymptotic variance using the delta method \cite{Casella2002}.  Note that these calculations omit any genes for which either observed read count is 0; these observations are generally removed by the trimming procedure.\\

The TMM normalization method attempts to derive a single value, $f$, such that $S_k = f \cdot S_{k\prime}$, where $S_k$ is the (unobserved) total count for sample $k$.  Unfortunately, this does not adequately account for the compositional nature of the data.  In particular, the authors assume that the expression levels will be the same among samples for the majority of genes (probes).  However, if we view the data as a composition, even if only a small number of genes differ with respect to expression level then all of the probes will differ with respect to what we actually observe, their \emph{proportion} of the sample. The authors correctly identify that some probes will be under-represented, even if their absolute expression remains unchanged, in samples with a large total RNA output as compared to samples with a smaller total RNA output.  This is because these probes will constitute a smaller proportion of the total number of reads, even if the absolute number will not have changed.\\

The authors do not explicitly reference Atichison, or any other compositional data publications, but their method relies heavily on these methods.  The TMM operation first converts all samples to the unit simplex by dividing each gene specific read count by the total counts for that sample ($Y_{gk}/N_k$). The full set of M-values calculated from these compositions represent the pertubation, $p$ from 1 sample to another.  However, the M-values are trimmed so $p$ is incomplete and we are now in a subcompositional sample space.  The assumption that most genes are not differentially expressed leads to the authors to use a weighted average of the M-values ($p_i$'s) to determine a single correction factor which can translate one composition to another.  This method, therefore, then assumes $p_i = p_j$ for \emph{most} of the genes $i$ and $j$ in the trimmed set.  \\

TMM normalization is not guaranteed to satisfy subcompositional coherance.  This is because changes in the counts of unmeasured genes can alter the... .  

%toy example here?





%\chapter{LASSO for Compositional Data}
%For large compositions use LASSO instead of traditional CoDa dimensional reduction techniques
%Use all parts for
%Use clr to  perform lasso on the D ratios

\chapter[Normalization]{Normalization of Compositional RNA Sequence Data}
%There are only d-1 degrees of freedom but which i to use ad the reference is not clear so alr or clr might be better.

\chapter[Correlaton]{An Alternative to Correlation for Evaluation of Reproducibility and Repeatability of Compositional Data}

%\chapter{Link between relative sensitivity and compositional data}


%\chapter{An R Package for Implementing Compositional Methods for RNA Sequence Data}



\printbibliography
\end{document}